packages/google_mobile_ads/lib/src/ad_containers.dart

```diff
@@ -763,6 +768,11 @@ class AdManagerInterstitialAd extends AdWithoutView {
   Future<void> show() {
     return instanceManager.showAdWithoutView(this);
   }
+
+  /// Enable / Disable immersive mode for the Ad.
+  Future<void> setImmersiveMode(bool immersiveModeEnabled) async {
+    return instanceManager.setImmersiveMode(this, immersiveModeEnabled);
```

Comment:
You can just define this method once in `AdWithoutView`.

Reply:
Also I think you make this no-op on iOS. Otherwise if a pub accidentally calls it they will get an error message.

```suggestion
    if (defaultTargetPlatform == TargetPlatform.iOS) {
      return Future.value();
    } else {
      return instanceManager.setImmersiveMode(this, immersiveModeEnabled);
    }
```